api_tests/btsieve/test.ts

```diff
@@ -73,7 +68,7 @@ describe("Test btsieve API", () => {
                     })
                     .then(res => {
                         res.should.have.status(201);
-                        location = res.headers.location;
+                        location = res.header.location;
                         location.should.be.a("string");
```

Comment:
🤡 Probably obsolete as a few lines up it is introduced as string.
Maybe check for not empty or a concrete location?
```suggestion
                         location.should.not.be.empty;
```